cnv/cnv_virtual_machines/cnv_vm_networking/virt-using-mac-address-pool-for-vms.adoc

```diff
@@ -0,0 +1,18 @@
+[id="virt-using-mac-address-pool-for-vms"]
+= Using a MAC address pool for virtual machines
+include::modules/cnv-document-attributes.adoc[]
+:context: virt-using-mac-address-pool-for-vms
+toc::[]
+
+:FeatureName: KubeMacPool
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+The _KubeMacPool_ component provides a MAC address pool service for additional NICs on virtual machines in designated namespaces. Enable a MAC address pool in a namespace by applying the KubeMacPool label to that namespace.
```

Comment:
please remove `additional`  in `The _KubeMacPool_ component provides a MAC address pool service for additional NICs...`. we don't want to be specific here.

Reply:
This was a ghost file (still uses the 'cnv-' prefix in folders and files) from when this PR was created (we had a PR merged a couple of weeks back that renamed all our files/folders). Thanks for pointing this out - I didn't realise this old assembly was still present but it's now been removed. The new assembly (virt/virtual_machines/vm_networking/virt-using-mac-address-pool-for-vms.adoc) reflects the update.